### PR TITLE
Current gmt alumni

### PR DESCRIPTION
**Pull request by ErunamoJAZZ**
### Description
Updated list of all Alumni in all languages.

Updated GMT list in English and Spanish.

### Status
Finished; Pending Review

### CODE REVIEW

File: wiki/People/Global_Moderation_Team/es.md [hunk 1]
```diff
@@ -21,45 +21,52 @@ Miembros del equipo
...
-| [Kobold84](https://osu.ppy.sh/u/3227533) | Ruso | Moderacion del chat |
-| [KSHR](https://osu.ppy.sh/u/409957) | Japonés | Moderacion del chat |
-| [Kurai](https://osu.ppy.sh/u/77089) | Francés, Ruso | Moderacion del chat |
-| [Marcin](https://osu.ppy.sh/u/722665) | Polaco | Moderación del foro, Desarrollo |
-| [MrSergio](https://osu.ppy.sh/u/2581696) | Italiano | Moderacion del chat |
-| [Nathanael](https://osu.ppy.sh/u/2295078) | Filipino | Moderación del foro, Moderacion del chat |
-| [Neil Watts](https://osu.ppy.sh/u/3048059) | Francés | Moderacion del chat |
-| [Nishiji](https://osu.ppy.sh/u/167629) | Turco | Moderacion del chat |
+| [-kevincela-](https://osu.ppy.sh/u/266596) | Italiano | |
+| [[ Another ]](https://osu.ppy.sh/u/3416573) | Húngaro | Chat moderation |
```
Comment (Repflez) [inline wiki/People/Global_Moderation_Team/es.md]: The entire file is using the English titles on the **Enfoque (Area of Focus)** column.

File: wiki/People/osu!_Alumni/ko.md [hunk 2]
```diff
@@ -5,47 +5,47 @@ osu! Alumni
 
 | Name | Epitaph |
```
Comment (libneko) [inline wiki/People/osu!_Alumni/ko.md]: Name = 이름, Epitaph = 비문

**Comment by libneko** [comment]
Translation of GMT list on wiki/People/Global_Moderation_Team/ko.md

```markdown
| 이름 | 언어 | 담당 영역 |
| ----- | --------- | ------------- |
| [-kevincela-](https://osu.ppy.sh/u/266596) | 이탈리아어 | |
| [[ Another ]](https://osu.ppy.sh/u/3416573) | 헝가리어 | 채팅 관리 |
| [Atrue](https://osu.ppy.sh/u/1758523) | 중국어 | 채팅 관리 |
| [Chaos](https://osu.ppy.sh/u/2628870) | | 채팅 관리 |
| [deadbeat](https://osu.ppy.sh/u/128370) | | osu!media 제작 |
| [Dntm8kmeeatu](https://osu.ppy.sh/u/5428812) | | 기술 지원 |
| [Flanster](https://osu.ppy.sh/u/447818) | 불가리아어 | osu!idol 호스팅, 채팅 관리 |
| [Gabe](https://osu.ppy.sh/u/654108) | 프랑스어 | 채팅 관리 |
| [Galkan](https://osu.ppy.sh/u/169570) | 폴란드어 | 채팅 관리 |
| [ghm12](https://osu.ppy.sh/u/2594229) | 포르투갈어 | 채팅 관리 |
| [heyronii](https://osu.ppy.sh/u/5642779) | 터키어 | 채팅 관리 |
| [IamKwaN](https://osu.ppy.sh/u/1856463) | 광둥어, 중국어 | 채팅 관리 |
| [juankristal](https://osu.ppy.sh/u/443656) | 스페인어 | 채팅 관리, 대회 |
| [Kobold84](https://osu.ppy.sh/u/3227533) | 러시아어 | 채팅 관리 |
| [Kurai](https://osu.ppy.sh/u/77089) | 프랑스어, 러시아어 | 채팅 관리 |
| [Kyubey](https://osu.ppy.sh/u/2195646) | 러시아어 | 채팅 관리 |
| [Marcin](https://osu.ppy.sh/u/722665) | 폴란드어 | 포럼 관리, 개발 |
| [MrSergio](https://osu.ppy.sh/u/2581696) | 이탈리아어 | 채팅 관리 |
| [n0ah](https://osu.ppy.sh/u/3086393) | 네덜란드어 | 채팅 관리 |
| [Nathanael](https://osu.ppy.sh/u/2295078) | 필리핀어 | 포럼 관리, 채팅 관리 |
| [Neil Watts](https://osu.ppy.sh/u/3048059) | 프랑스어 | 채팅 관리 |
| [Nyquill](https://osu.ppy.sh/u/682935) | 중국어 | osu!weekly |
| [Pawsu](https://osu.ppy.sh/u/2371454) |  | 채팅 관리, 기술 지원 |
| [Pereira006](https://osu.ppy.sh/u/537344) | 포르투갈어 | 채팅 관리 |
| [Pittigbassje](https://osu.ppy.sh/u/2167433) | 네덜란드어 | 채팅 관리 |
| [Repflez](https://osu.ppy.sh/u/201392) | 스페인어 | 채팅 관리, 위키 관리 |
| [ruexia](https://osu.ppy.sh/u/385069) | 한국어 | 채팅 관리 |
| [S o h](https://osu.ppy.sh/u/2234772) | 일본어 | 채팅 관리 |
| [Saten](https://osu.ppy.sh/u/444506) | 스웨덴어, 스페인어 | 채팅 관리 |
| [shARPII](https://osu.ppy.sh/u/776257) | 프랑스어 | 채팅 관리, 대회 |
| [Shiro](https://osu.ppy.sh/u/113005) | 프랑스어, 스페인어 | 채팅 관리 |
| [Shockwave](https://osu.ppy.sh/u/251631) | 스페인어 | 채팅 관리 |
| [Shurelia](https://osu.ppy.sh/u/3807986) | 인도네시아어 | 채팅 관리 |
| [Spkz](https://osu.ppy.sh/u/2964029) | 헝가리어 | 채팅 관리 |
| [Stefan](https://osu.ppy.sh/u/626907) | 독일어 | 포럼 관리, 채팅 관리 |
| [Sylphi](https://osu.ppy.sh/u/1399551) | 한국어 | 채팅 관리 |
| [Tasha](https://osu.ppy.sh/u/1031958) | | osu!media 제작 |
| [TheVileOne](https://osu.ppy.sh/u/278862) | | 기술 지원 |
| [ToGlette](https://osu.ppy.sh/u/1076236)| 한국어, 일본어 | 채팅 관리 |
| [Trosk-](https://osu.ppy.sh/u/3469385) | 스페인어 | 기술 지원 |
| [Ukami](https://osu.ppy.sh/u/820865) | 폴란드어 | 채팅 관리 |
| [Uni](https://osu.ppy.sh/u/617106) | 네덜란드어 | 채팅 관리 |
| [VeilStar](https://osu.ppy.sh/u/4255720) | | 기술 지원 |
| [WalterToro](https://osu.ppy.sh/u/5281416) | 스페인어 | 채팅 관리 |
| [Yason](https://osu.ppy.sh/u/2574392) | 폴란드어 | 채팅 관리 |
| [Zero__wind](https://osu.ppy.sh/u/1822830) | 중국어 | 채팅 관리 |
| [Zetera](https://osu.ppy.sh/u/587737) | 독일어 |  |
```

**Comment by libneko** [comment]
It seems that GMT list on other files which exist was not updated btw.

File: wiki/People/osu!_Alumni/pl.md [hunk 3]
```diff
@@ -6,67 +6,68 @@ Alumni osu!
```
Comment (TPGPL) [inline wiki/People/osu!_Alumni/pl.md]: translate Chat Moderator into Moderator czatu

File: wiki/People/osu!_Alumni/pl.md [hunk 4]
```diff
@@ -6,67 +6,68 @@ Alumni osu!
 
 | Nick | Zasługi |
 | ----- | ------- |
+| [-SiN-](https://osu.ppy.sh/u/10560) | Chat Moderator |
```
Comment (TPGPL) [inline wiki/People/osu!_Alumni/pl.md]: Moderator czatu

File: wiki/People/osu!_Alumni/pl.md [hunk 5]
```diff
@@ -6,67 +6,68 @@ Alumni osu!
...
 | [An64fan](https://osu.ppy.sh/u/38836)	| BAT |
 | [Andrea](https://osu.ppy.sh/u/33599) | GMT |
 | [Armin](https://osu.ppy.sh/u/105902)	| BAT |
 | [Asphyxia](https://osu.ppy.sh/u/1715720) | QAT |
-| [Atrue](https://osu.ppy.sh/u/1758523) | GMT |
 | [awp](https://osu.ppy.sh/u/2650) | GMT |
 | [Azure_Kite](https://osu.ppy.sh/u/66296) | MAT |
-| [Ballance](https://osu.ppy.sh/u/165946) | Moderator czatu |
+| [Bakari](https://osu.ppy.sh/u/597957) | GMT |
+| [Ballance](https://osu.ppy.sh/u/165946) | Chat Moderator |
```
Comment (TPGPL) [inline wiki/People/osu!_Alumni/pl.md]: Moderartor czatu

File: wiki/People/osu!_Alumni/pl.md [hunk 6]
```diff
@@ -6,67 +6,68 @@ Alumni osu!
...
 | [Cyclone](https://osu.ppy.sh/u/18589) | GMT |
 | [Cyril_Scarlet](https://osu.ppy.sh/u/108066) | MAT |
+| [DakeDekaane](https://osu.ppy.sh/u/1425253) | QAT |
 | [Dakeyrus](https://osu.ppy.sh/u/10441) | GMT |
 | [Dangaard](https://osu.ppy.sh/u/19488) | BAT |
 | [Darksonic](https://osu.ppy.sh/u/570042) | GMT |
 | [DawnII](https://osu.ppy.sh/u/8399) | BAT |
 | [deepsea](https://osu.ppy.sh/u/7824) | BAT |
-| [Derekku](https://osu.ppy.sh/u/91341) | Administracja |
+| [Derekku](https://osu.ppy.sh/u/91341) | Administration |
```
Comment (TPGPL) [inline wiki/People/osu!_Alumni/pl.md]: Administracja

File: wiki/People/osu!_Alumni/pl.md [hunk 7]
```diff
@@ -6,67 +6,68 @@ Alumni osu!
...
+| [Desperate-kun](https://osu.ppy.sh/u/2202645) | QAT |
 | [DiamondCrash](https://osu.ppy.sh/u/123790) | BAT |
 | [DJPop](https://osu.ppy.sh/u/2363) | BAT |
 | [dkun](https://osu.ppy.sh/u/154400) | BAT |
 | [Doddler](https://osu.ppy.sh/u/54) | BAT |
 | [Doomsday](https://osu.ppy.sh/u/18983) | BAT |
 | [Drafura](https://osu.ppy.sh/u/326099) | BAT |
-| [Dragvon](https://osu.ppy.sh/u/126927) | Moderator czatu |
+| [Dragvon](https://osu.ppy.sh/u/126927) | Chat Moderator |
```
Comment (TPGPL) [inline wiki/People/osu!_Alumni/pl.md]: Moderator czatu

File: wiki/People/osu!_Alumni/pl.md [hunk 8]
```diff
@@ -6,67 +6,68 @@ Alumni osu!
...
 | [drum drum](https://osu.ppy.sh/u/4435526) | GMT |
 | [Duoprism](https://osu.ppy.sh/u/7186) | BAT |
-| [Echo](https://osu.ppy.sh/u/431) | Tworzenie osu! |
+| [Echo](https://osu.ppy.sh/u/431) | osu!dev |
```
Comment (TPGPL) [inline wiki/People/osu!_Alumni/pl.md]: Programista osu!

File: wiki/People/osu!_Alumni/pl.md [hunk 9]
```diff
@@ -6,67 +6,68 @@ Alumni osu!
...
 | [EiJi](https://osu.ppy.sh/u/2024) | BAT |
-| [ethox](https://osu.ppy.sh/u/441380) | Moderator czatu |
+| [ErunamoJAZZ](https://osu.ppy.sh/u/1869764) | GMT |
+| [ethox](https://osu.ppy.sh/u/441380) | Chat Moderator |
```
Comment (TPGPL) [inline wiki/People/osu!_Alumni/pl.md]: Moderator czatu

File: wiki/People/osu!_Alumni/pl.md [hunk 10]
```diff
@@ -76,7 +77,7 @@ Alumni osu!
 | [Guy](https://osu.ppy.sh/u/91738) | QAT |
 | [h3k1ru](https://osu.ppy.sh/u/291211) | GMT |
 | [happy30](https://osu.ppy.sh/u/27767) | BAT |
-| [HeatKai](https://osu.ppy.sh/u/332555) | Moderator czatu |
+| [HeatKai](https://osu.ppy.sh/u/332555) | Chat Moderator |
```
Comment (TPGPL) [inline wiki/People/osu!_Alumni/pl.md]: Moderator czatu

File: wiki/People/osu!_Alumni/pl.md [hunk 11]
```diff
@@ -93,13 +94,14 @@ Alumni osu!
 | [Kharl](https://osu.ppy.sh/u/452) | BAT |
 | [kingcobra52](https://osu.ppy.sh/u/9934) | BAT |
 | [kingking9](https://osu.ppy.sh/u/1277097) | GMT |
-| [Kitokofox](https://osu.ppy.sh/u/1815420) | Moderator czatu |
+| [Kitokofox](https://osu.ppy.sh/u/1815420) | Chat Moderator |
```
Comment (TPGPL) [inline wiki/People/osu!_Alumni/pl.md]: Moderator czatu

File: wiki/People/osu!_Alumni/pl.md [hunk 12]
```diff
@@ -93,13 +94,14 @@ Alumni osu!
...
 | [Krah](https://osu.ppy.sh/u/1436748) | GMT |
 | [Krisom](https://osu.ppy.sh/u/99269) | BAT |
-| [KRZY](https://osu.ppy.sh/u/114017) | Moderator czatu |
+| [KRZY](https://osu.ppy.sh/u/114017) | Chat Moderator |
```
Comment (TPGPL) [inline wiki/People/osu!_Alumni/pl.md]: Moderator czatu

File: wiki/People/osu!_Alumni/pl.md [hunk 13]
```diff
@@ -93,13 +94,14 @@ Alumni osu!
...
+| [KSHR](https://osu.ppy.sh/u/409957) | GMT |
 | [Kylecito](https://osu.ppy.sh/u/488) | BAT |
 | [Kytoxid](https://osu.ppy.sh/u/98460) | BAT |
-| [La Cataline](https://osu.ppy.sh/u/301279) | Moderator czatu |
+| [La Cataline](https://osu.ppy.sh/u/301279) | Chat Moderator |
```
Comment (TPGPL) [inline wiki/People/osu!_Alumni/pl.md]: Moderator czatu

File: wiki/People/osu!_Alumni/pl.md [hunk 14]
```diff
@@ -121,28 +124,30 @@ Alumni osu!
...
 | [Melophobia](https://osu.ppy.sh/u/1077845) | BAT |
 | [minyeob](https://osu.ppy.sh/u/9207) | BAT |
+| [MMzz](https://osu.ppy.sh/u/128993) | QAT |
 | [Mogsworth](https://osu.ppy.sh/u/4018) | BAT |
 | [MOOMANiBE](https://osu.ppy.sh/u/8950) | BAT |
-| [Mr Color](https://osu.ppy.sh/u/116078) | GMT |
 | [mrtn](https://osu.ppy.sh/u/7013) | BAT |
 | [mtmcl](https://osu.ppy.sh/u/5960) | BAT |
-| [Mystyk](https://osu.ppy.sh/u/930165) | Moderator czatu |
+| [Mystyk](https://osu.ppy.sh/u/930165) | Chat Moderator |
```
Comment (TPGPL) [inline wiki/People/osu!_Alumni/pl.md]: Moderator czatu

File: wiki/People/osu!_Alumni/pl.md [hunk 15]
```diff
@@ -121,28 +124,30 @@ Alumni osu!
...
 | [narakucrimson](https://osu.ppy.sh/u/287468) | BAT |
 | [NatsumeRin](https://osu.ppy.sh/u/151679) | BAT |
 | [nekodex](https://osu.ppy.sh/u/102) | BAT |
+| [Nishiji](https://osu.ppy.sh/u/167629) | GMT |
 | [Niva](https://osu.ppy.sh/u/197805) | BAT |
 | [NoHitter](https://osu.ppy.sh/u/124455) | BAT |
 | [nuudles](https://osu.ppy.sh/u/21312) | osu!dev |
 | [Nymph](https://osu.ppy.sh/u/601990) | BAT |
-| [Oinari-sama](https://osu.ppy.sh/u/405508) | Moderator czatu i Członek Supportu |
+| [Oinari-sama](https://osu.ppy.sh/u/405508) | Chat Moderator and Support Team |
```
Comment (TPGPL) [inline wiki/People/osu!_Alumni/pl.md]: Moderator czatu i członek pomocy technicznej

File: wiki/People/osu!_Alumni/pl.md [hunk 16]
```diff
@@ -152,7 +157,7 @@ Alumni osu!
 | [SapphireGhost](https://osu.ppy.sh/u/388602) | BAT |
 | [Saturos](https://osu.ppy.sh/u/3781) | BAT |
 | [Seibei4211](https://osu.ppy.sh/u/31537) | BAT |
-| [senaya](https://osu.ppy.sh/u/304103) | Moderator czatu |
+| [senaya](https://osu.ppy.sh/u/304103) | Chat Moderator |
```
Comment (TPGPL) [inline wiki/People/osu!_Alumni/pl.md]: Moderator czatu

File: wiki/People/osu!_Alumni/pl.md [hunk 17]
```diff
@@ -167,25 +172,26 @@ Alumni osu!
...
 | [Tapdancingmetroid](https://osu.ppy.sh/u/1733) | BAT |
+| [Tari](https://osu.ppy.sh/u/1779172) | QAT |
 | [Teara](https://osu.ppy.sh/u/123491) | BAT |
 | [tensheapz](https://osu.ppy.sh/u/60) | BAT |
 | [theowest](https://osu.ppy.sh/u/60604) | GMT |
 | [tieff](https://osu.ppy.sh/u/89619) | BAT |
 | [TKS](https://osu.ppy.sh/u/940878) | BAT |
 | [tsububu](https://osu.ppy.sh/u/61) | BAT |
-| [UnderminE](https://osu.ppy.sh/u/444223) | Moderator czatu |
+| [UnderminE](https://osu.ppy.sh/u/444223) | Chat Moderator |
```
Comment (TPGPL) [inline wiki/People/osu!_Alumni/pl.md]: Moderator czatu

File: wiki/People/osu!_Alumni/pl.md [hunk 18]
```diff
@@ -167,25 +172,26 @@ Alumni osu!
...
 | [vytalibus](https://osu.ppy.sh/u/10028) | BAT |
 | [Where](https://osu.ppy.sh/u/549172) | GMT |
 | [whymeman](https://osu.ppy.sh/u/51994) | GMT |
-| [Winshley](https://osu.ppy.sh/u/864895) | Moderator czatu |
+| [Winshley](https://osu.ppy.sh/u/864895) | Chat Moderator |
```
Comment (TPGPL) [inline wiki/People/osu!_Alumni/pl.md]: Moderator czatu

File: wiki/People/osu!_Alumni/pl.md [hunk 19]
```diff
@@ -167,25 +172,26 @@ Alumni osu!
...
 | [wmfchris](https://osu.ppy.sh/u/7401) | BAT |
 | [Xgor](https://osu.ppy.sh/u/98661) | BAT |
 | [xierbaliti](https://osu.ppy.sh/u/34044) | BAT |
-| [XPJ38](https://osu.ppy.sh/u/273531) | Członek Supportu |
+| [XPJ38](https://osu.ppy.sh/u/273531) | Support Team |
```
Comment (TPGPL) [inline wiki/People/osu!_Alumni/pl.md]: Członek pomocy technicznej

**Comment by ErunamoJAZZ** [comment]
@libneko do you mean GMT list in other languages? or list in `Language_Moderators` and other folders?
Because I have no time to translate to languages that are not spanish, and because I have plans to update other list step to step.

**Comment by libneko** [comment]
@ErunamoJAZZ I mean, GMT list in the files of Global_Moderation_Team folder. I just thought It's necessary to update the page which exists, even if you do not translate.

**Comment by TPGPL** [comment]
What blocks you from updating every GMT list?

**Comment by ImMock** [comment]
Im making some fixes to, and im updating every list in all languages if he cant make them

**Comment by ErunamoJAZZ** [comment]
Thanks to @ImMock, all languages have the GMT list updated.

Let me know if you need any other change :)

File: wiki/People/Global_Moderation_Team/es.md (hunk 1 above)
Comment (ErunamoJAZZ) [reply wiki/People/Global_Moderation_Team/es.md]: It is actually fixed.

**Comment by libneko** [comment]
L23 ~ 26, L40, L43, L47, L58, L63: moderation -> Moderation, You need to capitalize the first letter like as other items.

**Comment by ErunamoJAZZ** [comment]
Done.

**Comment by Repflez** [comment]
Your PR has several commits with non-descriptive messages.

Please look into [changing your message to be more descriptive](https://help.github.com/articles/changing-a-commit-message/), and open a new PR once it has been fixed. This PR will be closed.

**Pull request closed without merging**